lib/ansible/executor/task_queue_manager.py

```diff
@@ -143,7 +143,7 @@ def load_callbacks(self):
         # first, load callbacks in the core distribution and configured callback paths
         for callback_plugin in callback_loader.all(class_only=True):
             callback_type = getattr(callback_plugin, 'CALLBACK_TYPE', '')
-            callback_needs_whitelist = getattr(callback_plugin, 'CALLBACK_NEEDS_WHITELIST', False)
+            callback_needs_whitelist = getattr(callback_plugin, 'CALLBACK_NEEDS_ENABLED', getattr(callback_plugin, 'CALLBACK_NEEDS_WHITELIST', False))
```

Comment:
If we want to be able to remove this one, we'll have to deprecate user usages of it- rather than adding a new term for something that now only exists in user plugins, should we just deprecate the underlying ability for automatic callback plugin loads altogether? eg, issue a deprecation warning on callbacks that load because they didn't set it, or that explicitly set it to `False`... I don't think we'd want to deprecate on the lack of the attr itself, since basically we'd be moving toward a future where it's just ignored anyway.

Reply:
i was leaving for backwards compat and giving authors the choice of changing name, unsure if we want to force the change on 3rd parties.